src/common/HaskellDo/Toolbar/View.hs

```diff
@@ -62,8 +65,20 @@ openProjectModal =
         div ! atr "class" "modal-footer" $
             div ! id "closeModalButton" $ noHtml
 
+convertToPDFModal :: Perch
+convertToPDFModal =
+    div ! id "convertToPDFModal" ! atr "class" "modal" $ do
+        div ! atr "class" "modal-content" $ do
+          h4 ("PDF saved on project path" :: String)
+
+convertToPDFModalFail :: Perch
+convertToPDFModalFail =
+    div ! id "convertToPDFModalFail" ! atr "class" "modal" $ do
+        div ! atr "class" "modal-content" $ do
+          h4 ("Error: wkhtmltopdf is not installed or a project has not been loaded" :: String)
```

Comment:
Not crucial, but Bootstrap allow for much richer modals with buttons ant that stuff to make users feel like they have some kind of control over their machines, I think that a `Dismiss` button would be a nice addition.

BTW @NickSeagull, which version of Bootstrap are you using? Bootstrap 4 recently reached beta status, which obviously means that it's production-ready xDD